tests/remote/remote_controller.pm

```diff
@@ -49,7 +50,13 @@ sub run {
     ensure_serialdev_permissions;
 
     if (check_var("REMOTE_CONTROLLER", "vnc")) {
-        select_console 'x11';
+        if(is_opensuse()) {
```

Comment:
If SLE is the exception I suggest to also make the code read like this. So better use `is_sle` with the explicit `send_key` in this branch.

Reply:
makes sense